test/support/utils.rb

```diff
@@ -29,8 +31,6 @@ def debug_code(program, boot_options: '-r debug/run', remote: true, &block)
         debug_on_unix_domain_socket
         debug_on_tcpip
       end
-
-      check_line_num!(program)
     end
 
     DEBUG_MODE = false
```

Comment:
I'd love to see it become a env var so I can always enable it locally. wdyt?

Reply:
Could you explain me more? I just moved `check_line_num` up. It is always enable, right?

Reply:
@st0012 
Could you explain me more? I just moved `check_line_num` up. It is always enable, right?

Reply:
I mean can we make this:

```ruby
DEBUG_MODE = ENV["RUBY_DEBUG_TEST_DEBUG_MODE"]
```

so I don't need to change the code to enabled it. like what you did and forgot to change it back (I've done this a lot too 😂 ).

Reply:
Oh, ok. I thought you were talking about `check_line_num`. I'll change `DEBUG_MODE`, too.

Reply:
fccf992